spec/fixtures/go-resolved-dependencies.json

```diff
@@ -3,104 +3,132 @@
     },
     {
       "Path": "dmitri.shuralyov.com/app/changes",
       "Main": "false",
       "Version": "v0.0.0-20180602232624-0a106ad413e3",
       "Indirect": "true",
-      "Scope": "runtime"
+      "Scope": "runtime",
+      "Replace": "<nil>"
     },
     {
       "Path": "dmitri.shuralyov.com/html/belt",
       "Main": "false",
       "Version": "v0.0.0-20180602232347-f7d459c86be0",
       "Indirect": "true",
-      "Scope": "runtime"
+      "Scope": "runtime",
+      "Replace": "<nil>"
     },
     {
       "Path": "dmitri.shuralyov.com/service/change",
       "Main": "false",
       "Version": "v0.0.0-20181023043359-a85b471d5412",
       "Indirect": "true",
-      "Scope": "runtime"
+      "Scope": "runtime",
+      "Replace": "<nil>"
     },
     {
       "Path": "dmitri.shuralyov.com/state",
       "Main": "false",
       "Version": "v0.0.0-20180228185332-28bcc343414c",
       "Indirect": "true",
-      "Scope": "runtime"
+      "Scope": "runtime",
+      "Replace": "<nil>"
     },
     {
       "Path": "git.apache.org/thrift.git",
       "Main": "false",
       "Version": "v0.0.0-20180902110319-2566ecd5d999",
       "Indirect": "true",
-      "Scope": "runtime"
+      "Scope": "runtime",
+      "Replace": "<nil>"
     },
     {
       "Path": "github.com/BurntSushi/toml",
       "Main": "false",
       "Version": "v0.3.1",
       "Indirect": "true",
-      "Scope": "runtime"
+      "Scope": "runtime",
+      "Replace": "<nil>"
     },
     {
       "Path": "github.com/Masterminds/semver",
       "Main": "false",
       "Version": "v1.5.0",
       "Indirect": "true",
-      "Scope": "runtime"
+      "Scope": "runtime",
+      "Replace": "<nil>"
     },
     {
       "Path": "github.com/Masterminds/semver/v3",
       "Main": "false",
       "Version": "v3.0.3",
       "Indirect": "true",
-      "Scope": "runtime"
+      "Scope": "runtime",
+      "Replace": "<nil>"
     },
     {
       "Path": "github.com/OneOfOne/xxhash",
       "Main": "false",
       "Version": "v1.2.2",
       "Indirect": "true",
-      "Scope": "runtime"
+      "Scope": "runtime",
+      "Replace": "<nil>"
     },
     {
       "Path": "github.com/ajg/form",
       "Main": "false",
       "Version": "v1.5.1",
       "Indirect": "true",
-      "Scope": "runtime"
+      "Scope": "runtime",
+      "Replace": "<nil>"
     },
     {
       "Path": "github.com/alecthomas/template",
       "Main": "false",
       "Version": "v0.0.0-20160405071501-a0175ee3bccc",
       "Indirect": "true",
-      "Scope": "runtime"
+      "Scope": "runtime",
+      "Replace": "<nil>"
     },
     {
       "Path": "github.com/alecthomas/units",
       "Main": "false",
       "Version": "v0.0.0-20151022065526-2efee857e7cf",
       "Indirect": "true",
-      "Scope": "runtime"
+      "Scope": "runtime",
+      "Replace": "<nil>"
     },
     {
       "Path": "github.com/stretchr/testify",
       "Main": "false",
       "Version": "v1.7.0",
       "Indirect": "false",
       "Scope": "test"
```

Comment:
NB: I've purposely left `"Replace"` out of this object to test that it's fine when it's missing.